server/helpers/getDefaultRoles.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+const Config = require('../../config');
+
+module.exports = Config.get('/role').map((role) => role.name);
```

Comment:
Please rename the file to `getRoleNames` and update the name across the codebase accordingly whenever you are importing this file.